update.sh

```diff
@@ -6,8 +6,8 @@
 # Description:
 #   An update bash script for engelsystem
 
-# Add upstream 
+# Add upstream
 git remote add upstream http://www.github.com/fossasia/engelsystem.git
```

Comment:
If a person Forks the project and makes some minor changes to it and uses it on his/her system, then remote would be required. 
In the case of forked repositories, "remote" will be different from "origin"